test/Serialization/PolynomialsSeries.jl

```diff
@@ -14,14 +15,14 @@ T = TropicalSemiring()
 
 cases = [
     (QQ, fmpq(3, 4), fmpq(1, 2), "Rationals"),
-    (ZZ, 3, 4, "Integers"),
```

Comment:
i figured these get checked already during some of the field extension checks, and that I have to start making these tests more efficient, I could just keep it in, for this pull requests since it seems a bit out of place

Reply:
Thanks for the explanation. No need to change this for me, I just wanted to understand.